topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.md

```diff
@@ -22,6 +22,13 @@ key_points:
 - Interpret (and correct if needed) GL(M)M results
 - Learn about and interpret common tests to evaluate the quality of your GL(M)M
 - Use an Essential Biodiversity Variables workflow
+tags:
+  - Species population EBV class
+  - Community composition EBV class
+  - EBV dataset
+  - EBV workflow
+  - data analysis
```

Comment:
is data analysis really useful here? (all tutorial here are data analysis yeah?)

just worried that if we do too many tags they become less useful, but your decision